test/document_stores/test_base.py

```diff
@@ -70,9 +70,9 @@ def test_write_documents(self, ds, documents):
         ds.write_documents(documents)
         docs = ds.get_all_documents()
         assert len(docs) == len(documents)
-        for i, doc in enumerate(docs):
-            expected = documents[i]
-            assert doc.id == expected.id
+        ids = set(doc.id for doc in documents)
+        expected_ids = set(doc.id for doc in docs)
```

Comment:
Just naming of variables: I guess the expected IDs come from `documents`, while the IDs we want to test come from `docs`.

```suggestion
        expected_ids = set(doc.id for doc in documents)
        ids = set(doc.id for doc in docs)
```

Reply:
good catch!